examples/kubernetes/1.11/cilium-crio-ds.yaml

```diff
@@ -41,8 +41,6 @@ spec:
               - "NET_ADMIN"
```

Comment:
Commit message typo: cri-o 1.11 --> k8s 1.11

Reply:
it's not a typo

Reply:
I am confused then. How do we know that the user is running with cri-o 1.11? The edited YAMLs were for k8s 1.11. Is the cri-o version directly tied to the K8s version, i.e. K8s 1.10 runs with cri-o 1.10, and cri-o 1.11 runs with K8s 1.11?

Reply:
yes https://github.com/kubernetes-sigs/cri-o#compatibility-matrix-cri-o---kubernetes-clusters

Reply:
Thanks, makes sense now.